docs/content-services/services/legal-hold.service.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: Legal Hold service
+Added: v6.10.0
+Status: Active
+Last reviewed: 2024-06-19
+---
+
+# [Legal Hold service](../../../lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts "Defined in legal-hold.service.ts")
+
+Manages holds for nodes.
+
+## Class members
+
+### Methods
+
+-   **getHolds**(filePlanId: `string`, options?: `ContentPagingQuery`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`Hold`](../../../lib/js-api/src/api/gs-core-rest-api/docs/Hold.md)`[]>`<br/>
+    Gets the list of holds for a node.
+    -   filePlanId: `string` - The identifier of a file plan. You can also use the -filePlan- alias
```

Comment:
Should not it be like: 
_filePlanId:_  ?

We have that in others md files I can see.

Reply:
the same for field below

Reply:
done